content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -293,4 +293,4 @@ kubectl get --raw "/api/v1/nodes/node-1.example/proxy/logs/?query=kubelet&patter
 * Read about [Contextual Logging](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/3077-contextual-logging)
 * Read about [deprecation of klog flags](https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components)
 * Read about the [Conventions for logging severity](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md)
-
+* Read about [Log Query](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/2258-node-log-query)
```

Comment:
We link to https://kep.k8s.io/2256 instead.

Reply:
I assume you meant  https://kep.k8s.io/2258. Fixed.